lib/tapioca/compilers/dsl/active_record_relations.rb

```diff
@@ -251,6 +251,15 @@ def generate
           sig { returns(String) }
           attr_reader :constant_name
 
+          sig { params(type: String).returns(String) }
```

Comment:
This inner class does not subclass from `Dsl::Base` so doesn't have access to this method. I added it here for the time being, since this inner class will go away very soon, and this method will go away with it.